README.md

```diff
@@ -15,6 +15,10 @@
 
 - [GitHub Deploy](https://jsug9.github.io/capstone-3/).
 
+## Presentation
+
+- To see the presentation in a video [click here](https://www.loom.com/share/2ec7c8700eb9424ca19512f292651b16).
+
 ## Getting Started
 
 To get a local copy of the project, follow these steps: 
```

Comment:
- [ ] It would be nicer to add some steps on how the users can run your project locally after cloning your repo (ex: npm install, npm run start ...)